sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetRowConverter.scala

```diff
@@ -139,7 +139,8 @@ private[parquet] class ParquetRowConverter(
     convertTz: Option[ZoneId],
     datetimeRebaseMode: LegacyBehaviorPolicy.Value,
     int96RebaseMode: LegacyBehaviorPolicy.Value,
-    updater: ParentContainerUpdater)
+    updater: ParentContainerUpdater,
+    parquetAccessByOrdinal: Boolean)
```

Comment:
nit: `parquetAccessByOrdinal` -> `accessByOrdinal`? parquet seems redundant in this context. Also please update the method comments with the new parameter.